tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -201,6 +205,20 @@ void test_compute_extrinsic_curvature_and_deriv_metric(const T& used_for_size) {
       gr::christoffel_first_kind(deriv_spatial_metric), inverse_spatial_metric);
 
   CHECK_ITERABLE_APPROX(christoffel_second_kind, christoffel_second_kind_test);
+
+  // Test Christoffel trace
+  const auto inverse_spacetime_metric =
+      gr::inverse_spacetime_metric(lapse, shift, inverse_spatial_metric);
+  const auto trace_christoffel = GeneralizedHarmonic::trace_christoffel(
+      spacetime_normal_one_form, spacetime_normal_vector,
+      inverse_spatial_metric, inverse_spacetime_metric, pi, phi);
+  tnsr::abb<T, Dim, Frame::Inertial> d4_spacetime_metric;
+  ::GeneralizedHarmonic::spacetime_derivative_of_spacetime_metric(
+      make_not_null(&d4_spacetime_metric), lapse, shift, pi, phi);
+  const auto expected_trace_christoffel =
+      trace_last_indices(gr::christoffel_first_kind(d4_spacetime_metric),
+                         inverse_spacetime_metric);
+  CHECK_ITERABLE_APPROX(trace_christoffel, expected_trace_christoffel);
```

Comment:
(Optional) If I'm paranoid, I could imagine this code and the trace function being broken together...maybe add a python/einsum test as well to catch such cases?